library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2046,22 +2046,37 @@ void libspdm_set_last_spdm_error_struct(void *context, libspdm_error_struct_t *l
 }
 
 /**
- * Initialize an SPDM context.
+ * Initialize an SPDM context, as well as all secured message contexts,
```

Comment:
You can delete all the Doxygen comments in .c files if they're present in a header. See #1273.

Reply:
Discussed, will leave Doxygen comments in .c files for widespread cleanup.